test/services/smart-fields-values-injector.js

```diff
@@ -0,0 +1,44 @@
+const chai = require('chai');
+const chaiSubset = require('chai-subset');
+const _ = require('lodash');
+const SmartFieldsValuesInjector = require('../../src/services/smart-fields-values-injector');
+const Schemas = require('../../src/generators/schemas');
+const usersSchema = require('../fixtures/users-schema.js');
+const addressesSchema = require('../fixtures/addresses-schema.js');
+
+const { expect } = chai;
+chai.use(chaiSubset);
+
+describe('Service > Smart Fields Values Injector', () => {
+
+  it('should work with a simple smart field', async () => {
+    Schemas.schemas = { users: usersSchema };
+    const record = { id: 123 };
+    const fieldsPerModel = { users: ['id', 'smart'] };
+    const injector = new SmartFieldsValuesInjector(record, 'users', fieldsPerModel);
+    await injector.perform();
+    expect(record).to.be.deep.equal({ id: 123, smart: { foo: 'bar' } });
+  });
+
+  it('should work without smart fields', async () => {
```

Comment:
[Code Style] More explicit test
```suggestion
    describe('without Smart Fields', () => {
      it('should not modify the record', async () => {
```

Reply:
I would move it as the first test to ease the test cases progression/readability.